interface/therapy_groups/index.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
```

Comment:
Needs a standard gpl header with your copyright and author tag. See here for example:
http://www.open-emr.org/wiki/index.php/How_to_Document_Your_Code_Properly#Page-Level_DocBlock